tests/Integration/Behaviour/Features/Scenario/Order/order_multi_invoices.feature

```diff
@@ -0,0 +1,698 @@
+    Then the available stock for product "Test Product A" should be 100
+    And the available stock for product "Test Product B" should be 100
+
+  Scenario: I add products twice in three different invoices and check that invoice totals and order totals are logical
+    When I generate invoice for "bo_order1" order
+    Then order "bo_order1" should have 1 invoices
+    When I add products to order "bo_order1" to first invoice and the following products details:
+      | name          | Test Product A |
+      | amount        | 3              |
+      | price         | 15             |
+    And I add products to order "bo_order1" with new invoice and the following products details:
+      | name          | Test Product A |
+      | amount        | 2              |
+      | price         | 15             |
+    And I add products to order "bo_order1" to second invoice and the following products details:
+      | name          | Test Product B |
+      | amount        | 1              |
+      | price         | 10             |
+    And I add products to order "bo_order1" with new invoice and the following products details:
+      | name          | Test Product B |
+      | amount        | 5              |
+      | price         | 10             |
+    Then order "bo_order1" should have 3 invoices
+    And order "bo_order1" should have 13 products in total
+    And order "bo_order1" should contain 2 products "Mug The best is yet to come"
+    And order "bo_order1" should contain 5 products "Test Product A"
+    And order "bo_order1" should contain 6 products "Test Product B"
+    And the available stock for product "Test Product A" should be 95
+    And the available stock for product "Test Product B" should be 94
+    And first invoice from order "bo_order1" should contain 2 products "Mug The best is yet to come"
+    And first invoice from order "bo_order1" should contain 3 products "Test Product A"
+    And first invoice from order "bo_order1" should have following details:
+      | total_products          | 68.800 |
+      | total_products_wt       | 72.930 |
+      | total_discount_tax_excl | 0.0    |
+      | total_discount_tax_incl | 0.0    |
+      | total_paid_tax_excl     | 75.80  |
+      | total_paid_tax_incl     | 80.35  |
+      | total_shipping_tax_excl | 7.0    |
+      | total_shipping_tax_incl | 7.42   |
+    And second invoice from order "bo_order1" should contain 2 products "Test Product A"
+    And second invoice from order "bo_order1" should contain 1 products "Test Product B"
+    And second invoice from order "bo_order1" should have following details:
+      | total_products          | 40.000 |
+      | total_products_wt       | 42.400 |
+      | total_discount_tax_excl | 0.0    |
+      | total_discount_tax_incl | 0.0    |
+#      | total_paid_tax_excl     | 47.00  |
+#      | total_paid_tax_incl     | 49.82  |
+#      | total_shipping_tax_excl | 7.0    |
+#      | total_shipping_tax_incl | 7.42   |
```

Comment:
Why is this commented ?

Reply:
This big behat feature comes from a bigger PR (see description) where everything was fixed But the PR is too big and the technical solution to fix shipping calculations and discount on invoices has not been validated by Pablo

So the idea of this PR is to integrate all the expected scenarios as a reference but some are commented because they are not fixed yet in this one, they will be uncommented in future PR that fix the bugs